src/utils/extensionUtils.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Event, EventEmitter, extensions } from "vscode";
+
+export class ExtensionUtils {
+    private extensionName: string;
+    private readonly extensionInstalledEmitter: EventEmitter<boolean> = new EventEmitter<boolean>();

```

Comment:
I believe an `EventEmitter<T>` is disposable, which means this type must also be disposable (and call the emitter's `dispose()` function).